lib/galaxy/managers/workflows.py

```diff
@@ -90,9 +90,14 @@ def check_security(self, trans, has_workflow, check_ownership=True, check_access
         if not check_ownership or check_accessible:
```

Comment:
It seems like we should allow this I guess (alternative we could copy to the workflow as one you do own). Hopefully they at least don't run when you do create them 😓?